db/types/categories.py

```diff
@@ -10,8 +10,6 @@
     *STRING_TYPES,
     PostgresType.CHAR,
     PostgresType.NAME,
-    MathesarCustomType.URI,
-    MathesarCustomType.EMAIL,
```

Comment:
Removing these with no other changes results in breaking searching for emails and URLs in the record selector.